client/src/pages/Courses/EmpyCourse.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Flex, Text } from "@chakra-ui/core";
+import { colors as c } from "../../colors";
+
+const EmptyCourse = () => {
```

Comment:
Can we rename this file name to `EmptyCourse` (not Empy hehe)

Reply:
done